src/TaggedInput.jsx

```diff
@@ -269,7 +269,13 @@ var TaggedInput = React.createClass({
     if (tagText && tagText.length > 0) {
       trimmedText = tagText.trim();
       if (p.unique) {
-        duplicateIndex = this.state.tags.indexOf(trimmedText);
+
+        // not a boolean, it's a function
+        if (p.unique !== true) {
```

Comment:
if u.unique is false then it wouldn't enter the if on line 271. I do like doing typeof in it's place though, so I will fix it.